tests/test_samples.py

```diff
@@ -1245,7 +1245,7 @@ def test_samples_dot_plot():
     assert len(axes) == 2
 
     axes = samples.plot.kde_2d('x0', 'x1')
-    assert len(axes.collections) == 5
+    assert len(axes.collections) == 2
```

Comment:
This test will fail on mpl<3.8. Is this a test we really need to do? The number of collections feels like matplotlib just doing matplotlib things

Reply:
I agree (although 2 makes more sense here than 5...). Is there anything explicit we can test to confirm that it has plotted something?

Reply:
Check it's more than 0?

Reply:
That would do